src/components/DashboardHome/DashboardHome.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import Hero from '../Hero/Hero';
+import CourseSection from '../CourseSection/CourseSection';
+import InProgressCourseCard from '../CourseCard/InProgressCourseCard';
+import UpcomingCourseCard from '../CourseCard/UpcomingCourseCard';
+import CompletedCourseCard from '../CourseCard/CompletedCourseCard';
+
+import sampleApiResponse from './sampleApiResponse';
+
+import './DashboardHome.scss';
+
+class DashboardHome extends React.Component {
+  state = {
+    courses: {
+      'in-progress': [],
+      upcoming: [],
+      completed: [],
+    },
+    // Temporarily adds `isProgramEnrollmentsLoading` to state to simulate loading
+    // state. Once we are actually calling the API and have Redux set up, this should
+    // come from props instead of state.
+    isProgramEnrollmentsLoading: true,
+  };
+
+  componentDidMount() {
+    this.groupCourseEnrollmentsByStatus();
+
+    // Temporarily simulate loading by resetting `isProgramEnrollmentsLoading` to false
+    // after 2 seconds.
+    setTimeout(() => {
+      this.setState({
+        isProgramEnrollmentsLoading: false,
+      });
+    }, 2000);
+  }
+
+  groupCourseEnrollmentsByStatus = () => {
+    const { course_runs: courseRuns } = sampleApiResponse;
+
+    this.setState({
+      courses: {
+        'in-progress': courseRuns.filter(courseRun => courseRun.status === 'in-progress'),
+        upcoming: courseRuns.filter(courseRun => courseRun.status === 'upcoming'),
+        completed: courseRuns.filter(courseRun => courseRun.status === 'completed'),
+      },
+    });
+  }
+
+  render() {
+    const { courses, isProgramEnrollmentsLoading } = this.state;
+
+    return (
+      <>
```

Comment:
what is this?

Reply:
It's the shorthand for a React fragment, instead of writing `<React.Fragment>` or `<Fragment>`.